horace_core/sqw/axes_blocks/@cylinder_axes/private/set_angles_in_rad_.m

```diff
@@ -0,0 +1,47 @@
+function obj = set_angles_in_rad_(obj,val)
```

Comment:
This check is performed in `check_combo_args_` when multiple properties are analyzed all togheter. You should not do this check here, because you may want to set different ranges while setting different property.